Gemfile.lock

```diff
@@ -1,12 +1,11 @@
+PATH
```

Comment:
This lock file contains a too big amount of changes. I don't any reason for doing such changes.

Reply:
Addressed